daprdocs/content/en/developing-applications/building-blocks/pubsub/howto-route-messages.md

```diff
@@ -238,7 +266,213 @@ $app->start();
+
+- Constraints:
+  - OPTIONAL
+  - If present, MUST adhere to the format specified in
+    [RFC 2046](https://tools.ietf.org/html/rfc2046)
+- For Media Type examples see
+  [IANA Media Types](http://www.iana.org/assignments/media-types/media-types.xhtml)
+
+#### dataschema
+
+- Type: `URI`
+- Description: Identifies the schema that `data` adheres to. Incompatible
+  changes to the schema SHOULD be reflected by a different URI. See
+  [Versioning of CloudEvents in the Primer](primer.md#versioning-of-cloudevents)
+  for more information.
+- Constraints:
+  - OPTIONAL
+  - If present, MUST be a non-empty URI
+
+#### subject
+
+- Type: `String`
+- Description: This describes the subject of the event in the context of the
+  event producer (identified by `source`). In publish-subscribe scenarios, a
+  subscriber will typically subscribe to events emitted by a `source`, but the
+  `source` identifier alone might not be sufficient as a qualifier for any
+  specific event if the `source` context has internal sub-structure.
+
+  Identifying the subject of the event in context metadata (opposed to only in
+  the `data` payload) is particularly helpful in generic subscription filtering
+  scenarios where middleware is unable to interpret the `data` content. In the
+  above example, the subscriber might only be interested in blobs with names
+  ending with '.jpg' or '.jpeg' and the `subject` attribute allows for
+  constructing a simple and efficient string-suffix filter for that subset of
+  events.
+
+- Constraints:
+  - OPTIONAL
+  - If present, MUST be a non-empty string
+- Example:
+  - A subscriber might register interest for when new blobs are created inside a
+    blob-storage container. In this case, the event `source` identifies the
+    subscription scope (storage container), the `type` identifies the "blob
+    created" event, and the `id` uniquely identifies the event instance to
+    distinguish separate occurrences of a same-named blob having been created;
+    the name of the newly created blob is carried in `subject`:
+    - `source`: https://example.com/storage/tenant/container
+    - `subject`: mynewfile.jpg
+
+#### time
+
+- Type: `Timestamp`
+- Description: Timestamp of when the occurrence happened. If the time of the
+  occurrence cannot be determined then this attribute MAY be set to some other
+  time (such as the current time) by the CloudEvents producer, however all
+  producers for the same `source` MUST be consistent in this respect. In other
+  words, either they all use the actual time of the occurrence or they all use
+  the same algorithm to determine the value used.
+- Constraints:
+  - OPTIONAL
+  - If present, MUST adhere to the format specified in
+    [RFC 3339](https://tools.ietf.org/html/rfc3339)
+
+{{% alert title="Limitation" color="warning" %}}
+Currently, comparisons to time (e.g. before or after "now") are not supported.
+{{% /alert %}}
 
 ## Next steps
```

Comment:
Can you add a link to an embedded video of the community call at the top of the page. Here is the specific link with the right time https://www.youtube.com/watch?v=QqJgRmbH82I. See an example of this here https://docs.dapr.io/operations/configuration/invoke-allowlist/

Effectively add this line
"Watch this video [embedded link] on how to use message routing with pub/sub."

Reply:
Finished adding my comments above.

Reply:
@msfussell I added this under "Next steps" since this is where you made the comment.  Other pages have it at the top after the intro paragraph. Let me know it you would prefer it there.